api.js

```diff
@@ -194,8 +209,10 @@ function handlePaths(files) {
 	if (files.length === 0) {
 		files = [
 			'test.js',
+			'**/test.js',
 			'test-*.js',
-			'test/*.js'
+			'**/test-*.js',
+			'test'
```

Comment:
Actually, the change to this array is wrong.

It should match `test.js` and `test-*.js` in root, and any JS file recursively in `test` folder as long as it's not prefixed with an underscore.